gk/main.c

```diff
@@ -359,13 +358,15 @@ gk_process_request(struct flow_entry *fe, struct ipacket *packet,
 
 	/* The assigned priority is @priority. */
 
+	struct grantor_entry *grantor = choose_grantor_per_flow(fe);
```

Comment:
Style: declare `struct grantor_entry *grantor;` at the beginning of the function.